receiver/prometheusexecreceiver/receiver_test.go

```diff
@@ -113,3 +120,663 @@ func assertTwoUniqueValuesScraped(t *testing.T, metricsSlice []pdata.Metrics) {
 
 	assert.Fail(t, "All %v scraped values were non-unique", len(metricsSlice))
 }
+
+func TestGetReceiverConfig(t *testing.T) {
```

Comment:
`TestGetReceiverConfig` & `TestGetSubprocessConfig` both include both `wantReceiverConfig` & `wantSubprocessConfig` but only assert against one of these.

You could remove the redundant expected values, but I'd probably recommend combining these tests instead.